pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -128,9 +129,12 @@ func RunNewBuild(fullName string, f *clientcmd.Factory, out io.Writer, in io.Rea
 	for _, item := range result.List.Items {
 		switch t := item.(type) {
 		case *buildapi.BuildConfig:
-			fmt.Fprintf(c.Out(), "A build configuration was created - you can run `%s start-build %s` to start it.\n", fullName, t.Name)
+			fmt.Fprintf(c.Out(), "Build configuration %q created and build triggered.\n", t.Name)
 		}
 	}
+	if len(result.List.Items) > 0 {
+		fmt.Fprintf(c.Out(), "Run '%s %s' to check the progress.\n", fullName, StatusRecommendedName)
```

Comment:
I feel `oc logs bc/name` (once https://github.com/openshift/origin/pull/4997 lands) or `oc get builds/buildName -w` are more appropriate than `oc status` for a running build.